src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF1012.MessageArgumentsMustBeConsistentWithRule_Valid.sarif

```diff
@@ -0,0 +1,101 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "SARIF Functional Testing",
+          "version": "1.2.3",
+          "rules": [
```

Comment:
>rules [](start = 11, length = 5)

Use simple test rules instead of real ones (`"TST1001"`). And construct the messages so the do not cause "high quality message" errors. For example: `"This is a test with an enquoted placeholder '{0}'."`